src/rapids_dependency_file_generator/rapids_dependency_file_validator.py

```diff
@@ -1,14 +1,16 @@
 """Logic for validating dependency files."""
 
+import importlib.resources
 import json
 import sys
 import textwrap
 
 import jsonschema
-import pkg_resources
 from jsonschema.exceptions import best_match
 
-SCHEMA = json.loads(pkg_resources.resource_string(__name__, "schema.json"))
+SCHEMA = json.loads(
+    importlib.resources.files(__package__).joinpath("schema.json").read_bytes()
+)
```

Comment:
In case anyone else comes to look at this PR, this change aligns with the official recommendation: https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-string

Reply:
Oh nice! I googled for a migration guide but I just guessed what code to use and got it right. 🏆